lib/rules/unsafe-to-chain-command.js

```diff
@@ -129,7 +129,7 @@ const isRootCypress = (node) => {
  */
 const isActionUnsafeToChain = (node, additionalMethods = []) => {
   const unsafeActionsRegex = new RegExp([
-    ...unsafeToChainActions,
+    ...unsafeToChainActions.map((action) => `^${action}$`),
```

Comment:
@gkorakas-eli This is nicer than the very specific hotfix I put forwards in https://github.com/cypress-io/eslint-plugin-cypress/pull/142 - I missed that this PR was open when making that PR to illustrate the problem.

If you pull in main now and revert the change I made in this file, but leave the tests I added, this looks like a better approach to me 👍 

WDYT @nagash77?

Reply:
LGTM - what do you think @nagash77?

Reply:
cc: @passbyval

Reply:
LGTM!